requirements.in

```
@@ -4,7 +4,7 @@ django-enumfields
 django-environ
 django-etuovi
 django-oikotie
-django-helusers
+django-helusers==0.13.0
```

```suggestion
django-helusers~=0.13
```

for allowing minor version increases (`>= 0.13, == 0.*`)